scrap.js

```diff
@@ -17,19 +19,82 @@ async function fetchLeaders(id) {
   return leaders
 }
 
-async function fetchOrgsWithLeaders() {
+async function searchGitHubOrgs(query) {
+  const token = process.env.GH_TOKEN
+  const res = await fetch(
+    `https://api.github.com/search/users?q=${query}%20type:org`,
+    {
+      headers: token ? { Authorization: `token ${token}` } : {}
+    }
+  )
+  const resp = await res.json()
+  const { items } = resp
```

Comment:
`const { items } = await res.json()`
is 1 line shorter.

Reply:
Yep, sorry, some debug code I forgot to get rid of!